lib/mongoid/criteria/includable.rb

```diff
@@ -0,0 +1,82 @@
+# encoding: utf-8
+module Mongoid
+  class Criteria
+
+    # Module providing functionality for parsing (nested) inclusion definitions.
+    module Includable
+
+      # Add an inclusion definition to the list of inclusions for the criteria.
+      #
+      # @example Add an inclusion.
+      #   criteria.add_inclusion(Person, :posts)
+      #
+      # @param [ Class, String, Symbol ] k The class or string/symbol of the class name.
+      # @param [ Symbol ] relation The relation.
+      #
+      # @raise [ Errors::InvalidIncludes ] If no relation is found.
+      #
+      # @since 5.1.0
+      def add_inclusion(k, relation)
```

Comment:
Style point--better to use `key` instead of `k`, and in other places `relation` instead of `r`, etc. which seems to be the standard throughout this repo.